.github/workflows/main.yml

```
@@ -92,3 +92,28 @@ jobs:
       # The Windows CI transforms the `c-library` symlink into a real directory, modifying the Git
       # state, so we ignore these changes with `--allow-dirty` here.
       run: scripts/verify-publish.sh --allow-dirty
+
+  python_build:
+    name: Check Python bindings
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: ubuntu-latest
+          - os: windows-latest
+    runs-on: ${{ matrix.os }}
+
+    defaults:
+      run:
+        working-directory: python
+
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - name: Build and Install ittapi
+        run: |
+          python3 -m pip install .
+      - name: Run unit-test
+        run: |
+          python3 -m unittest discover -s utest -t utest
```

```suggestion
  python_build:
    name: Check Python bindings
    runs-on: ${{ matrix.os }}
    strategy:
      fail-fast: false
      matrix:
        include:
          - os: ubuntu-latest
          - os: windows-latest
    defaults:
      run:
        working-directory: python
    steps:
      - name: Checkout sources
        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
      - name: Build and install ittapi package
        run: python -m pip install .
      - name: Run unit tests
        run: python -m unittest discover -s utest -t utest
```
I suggest the following:
- move "runs-on" section on the top
- delete empty lines
- adapt "run" to single-line commands by removing a pipe "|"
- steps renaming
- switch use default system python